galite-tests/src/test/kotlin/org/kopi/galite/demo/command/CommandR.kt

```diff
@@ -0,0 +1,169 @@
+
+  val action = menu("Action")
+
+  val csv = actor(
+          ident = "CSV",
+          menu = action,
+          label = "CSV",
+          help = "Obtenir le format CSV",
+  ) {
+    key = Key.F8          // key is optional here
+    icon = "export"  // icon is optional here
+  }
+
+  val xls = actor(
+          ident = "XLS",
+          menu = action,
+          label = "XLS",
+          help = "Obtenir le format Excel (XLS)",
+  ) {
+    key = Key.SHIFT_F8          // key is optional here
+    icon = "export"  // icon is optional here
+  }
+
+  val xlsx = actor(
+          ident = "XLSX",
+          menu = action,
+          label = "XLSX",
+          help = "Obtenir le format Excel (XLSX)",
+  ) {
+    key = Key.SHIFT_F8          // key is optional here
+    icon = "export"  // icon is optional here
+  }
+
+  val pdf = actor(
+          ident = "PDF",
+          menu = action,
+          label = "PDF",
+          help = "Obtenir le format PDF",
+  ) {
+    key = Key.F9          // key is optional here
+    icon = "export"  // icon is optional here
+  }
+
+  val cmdCSV = command(item = csv) {
+    action = {
+      model.export(VReport.TYP_CSV)
+    }
+  }
+
+  val cmdPDF = command(item = pdf) {
+    action = {
+      model.export(VReport.TYP_PDF)
+    }
+  }
+
+  val cmdXLS = command(item = xls) {
+    action = {
+      model.export(VReport.TYP_XLS)
+    }
+  }
+
+  val cmdXLSX = command(item = xlsx) {
+    action = {
+      model.export(VReport.TYP_XLSX)
+    }
+  }
+
+  val numCmd = field(Domain<Int>(25)) {
+    label = "Command number"
+    help = "The command number"
+    align = FieldAlignment.LEFT
+
+  }
+
+  val idClt = field(Domain<Int>(25)) {
+    label = "Client ID"
+    help = "The command client ID"
+    align = FieldAlignment.LEFT
+
+  }
+  val dateCmd = field(Domain<String>(50)) {
+    label = "Command date"
+    help = "The command date"
+    align = FieldAlignment.LEFT
+  }
+
+  val paymentMethod = field(Payment) {
+    label = "Payment method"
+    help = "The payment method"
+    align = FieldAlignment.LEFT
+  }
+
+  val statusCmd = field(CommandStatus) {
+    label = "Command status"
+    help = "The command status"
+    align = FieldAlignment.LEFT
+  }
+
+  object Payment : CodeDomain<String>() {
+    init {
+      "cash" keyOf "cash"
+      "check" keyOf "check"
+      "bank card" keyOf "bank card"
+    }
+  }
+
+  object CommandStatus : CodeDomain<String>() {
+    init {
+      "In preparation" keyOf "in preparation"
+      "available" keyOf "available"
+      "delivered" keyOf "delivered"
+      "canceled" keyOf "canceled"
+    }
+  }
+
+  val q = Command.selectAll()
+
+  init {
+    transaction {
+      q.forEach { result ->
+        add {
+          this[numCmd] = result[Command.numCmd]
+          this[idClt] = result[Command.idClt]
+          this[dateCmd] = result[Command.dateCmd]
+          this[paymentMethod] = result[Command.paymentMethod]
+          this[statusCmd] = result[Command.statusCmd]
```

Comment:
check data selected we have ("en cours") also we have ("carte", "espece") , so why paymentMethod & statusCmd accept value out of their domain?